### DISCUSSION TITLE
[egui_wgpu question] Texture Managed(0) used but not live error

### DISCUSSION THREAD

**Discussion by ElhamAryanpur** [emilk/egui#1838] [Q&A]
Hey there! I've been trying to add egui to a custom engine that have wgpu as base, I've been getting that texture managed 0 error at startup... I've followed almost exactly line by line of https://github.com/hasenbanck/egui_example

**Comment by batatoid-dev** [comment]
I'm also getting the same error on my project, have you solved it in any way by now?

**Reply by ElhamAryanpur** [reply]
Hey, well yes but I've moved to depend on egui provided crates.

[Current implementation](https://github.com/AryanpurTech/BlueEngineEGUI/blob/master/src/lib.rs)

[old implementation using this crate](https://github.com/AryanpurTech/BlueEngineEGUI/blob/1025dd60fcdde2ad66779938b2e0c19effecd3f3/src/lib.rs)

**Comment by PWhiddy** [comment]
For anyone else encountering this issue, I was missing updating the textures based on the full_output:
```rust
for (id, image_delta) in &full_output.textures_delta.set {
    egui_renderer.update_texture(
        &device, 
        &queue, 
        *id, 
        image_delta
    );
}
```
